paper/paper.md

```diff
@@ -0,0 +1,170 @@
+---
+title: 'MNE-BIDS'
+tags:
+  - Python
+  - BIDS
+  - MNE
+  - neuroimaging
+  - MEG
+  - EEG
+  - iEEG
+authors:
+  - affiliation: 1
+    name: Stefan Appelhoff
+    orcid: 0000-0001-8002-0877
+  - affiliation: 2
+    name: Mainak Jas
+    orcid: 0000-0002-3199-9027
+  - affiliation: 3
+    name: monkeyman192
+    orcid: xxxx-xxxx-xxxx-xxxx
+  - affiliation: 4
+    name: Teon L Brooks
+    orcid: 0000-0001-7344-3230
+  - affiliation: 5
+    name: Marijn van Vliet
+    orcid: 0000-0002-6537-6899
+  - affiliation: 6
+    name: Romain Quentin
+    orcid: 0000-0001-7659-3605
+  - affiliation: 7
+    name: Chris Holdgraf
+    orcid: 0000-0002-2391-0678
+  - affiliation: 8
+    name: dnacombo
+    orcid: xxxx-xxxx-xxxx-xxxx
+  - affiliation: 9
+    name: Ezequiel Mikulan
+    orcid: 0000-0001-7259-6120
+  - affiliation: 10
+    name: ktavabi
+    orcid: xxxx-xxxx-xxxx-xxxx
+  - affiliation: 11
+    name: hoechenberger
+    orcid: xxxx-xxxx-xxxx-xxxx
+  - affiliation: 12
+    name: dominikwelke
+    orcid: xxxx-xxxx-xxxx-xxxx
+  - affiliation: 13
+    name: Clemens Brunner
+    orcid: 0000-0002-6030-2233
+  - affiliation: 14
+    name: Alexander P Rockhill
+    orcid: 0000-0003-3868-7453
+  - affiliation: 15
+    name: Alexandre Gramfort
+    orcid: 0000-0001-9791-4404
+affiliations:
+- index: 1
+  name: Center for Adaptive Rationality, Max Planck Institute for Human Development, Berlin, Germany
+- index: 2
+  name: Athinoula A. Martinos Center for Biomedical Imaging, Massachusetts General Hospital, Charlestown, MA, USA
+- index: 3
+  name:
+- index: 4
+  name: Mozilla
+- index: 5
+  name: Department of Neuroscience and Biomedical Engineering, Aalto University, Espoo, Finland
+- index: 6
+  name: Human Cortical Physiology and Neurorehabilitation Section, NINDS, NIH, Bethesda, Maryland 20892
+- index: 7
+  name: UC Berkeley, Project Jupyter
+- index: 8
+  name:
+- index: 9
+  name: Department of Biomedical and Clinical Sciences 'L. Sacco', University of Milan, Milan, Italy
+- index: 10
+  name:
+- index: 11
+  name:
+- index: 12
+  name:
+- index: 13
+  name: Institute of Psychology, University of Graz, Austria
+- index: 14
+  name: University of Oregon, Eugene OR, USA
+- index: 15
+  name: Université Paris-Saclay, Inria, CEA, Palaiseau, France
+date: 28 October 2019
+bibliography: paper.bib
+---
+
+# Summary
+
+With the emergence of the Brain Imaging Data Structure
+(``BIDS``; [@Gorgolewski2016]) in 2016, the scientific community received a
+standard to organize and share data in the broad domain of neuroscience.
+
+Originally limited to MRI data types, BIDS is continuously being extended and
+by now also supports other neuroimaging modalities such as MEG [@Niso2018],
+EEG [@Pernet2019], and iEEG [@Holdgraf2019].
+
+BIDS prescribes how complex experiment data should be structured and which
+metadata should be encoded next to the raw data. This set of guidelines
+allows users to reap several benefits, such as:
```

Comment:
~~allows users to reap several benefits, such as:~~
provides the following benefits:

Reply:
Then I'd capitalize the first letter of each bullet point.